index.js

```diff
@@ -0,0 +1,2 @@
+console.log('index.js')
```

Comment:
```suggestion
console.log('jonsoku.js')
```
이 코드로 바꾸면 더 멋질거 같아요 !!

Reply:
바꿨다!